lib/calculate-reward.js

```diff
@@ -18,13 +18,12 @@ const calculateReward = async (validators = [], amount, timePeriodValue, timePer
 	// 21900 = number of eras in 15 years (365 * 4 * 15)
 	if (timePeriodInEras > 21900) throw new Error('15 years are enough to make you rich, please adjust the time period.');
 
-	let totalReward = 0;
-	validators.forEach(v => {
+	const totalReward = validators.reduce((tr, v) => {
 		const stakeFraction = amountPerValidator / (amountPerValidator + v.totalStake);
-		const reward = Math.max(v.estimatedPoolReward - v.commission, 0) * stakeFraction;
+		const reward = Math.max(v.estimatedPoolReward - (v.commission / 100 * v.estimatedPoolReward) , 0) * stakeFraction;
```

Comment:
@akshatbhargava123 Why do we even proceed forward if we're getting a negative value for `v.estimatedPoolReward - (v.commission / 100 * v.estimatedPoolReward)? The output will always be incorrect in such a scenario, and in such a case there's clearly a mistake on our part - i.e. the user shouldn't receive wrong info.

Reply:
discussed, will remove it now.